### ISSUE TITLE
function destroyBlock in explosions

### ISSUE THREAD

**Issue by Rodrigo294X** [zhuowei/MCPELauncher#930]
In an explosion the hook "destroy Block" should not detect the broken blocks?

**Comment by WIPocket** [comment]
No. destroyBlock() only triggers when a player breakes it. But there is a hook to detect explozions
